Makefile

```
@@ -51,6 +51,7 @@ plan:
 
 all: $(addsuffix /all,$(MAKEDIRS))
 
+init: ;
```

```suggestion
init:
```
I think it's fine to just leave it empty